opentaxii/utils.py

```diff
@@ -261,9 +249,7 @@ def sync_accounts(server, accounts):
     missing_usernames = (
         set(existing_by_username.keys()) - set(defined_by_username.keys()))
     for username in missing_usernames:
-        manager.delete_account(username)
-        deleted_counter += 1
-        log.info("sync_accounts.deleted", username=username)
+        pass
```

Comment:
these should not be removed, right? ↑

Reply:
@traut Yes; my bad. Didn't realize those were gone